doc/flame/game.md

```diff
@@ -1,3 +1,46 @@
+# GameWidget
+
+A `GameWidget` is the Flutter `Widget` that is used to insert a `Game` inside the flutter widget
+tree.
+
+It can directly receive a `Game` instance by using its default construct or it can receive a
+`GameCreate` function on the `controlled` constructor that will be used to create the game
+once the `GameWidget` is inserted on the widget tree.
```

Comment:
```suggestion
It can directly receive a `Game` instance by using its default constructor or it can receive a
`GameCreate` function on the `controlled` constructor that will be used to create the game
once the `GameWidget` is inserted in the widget tree.
```

Reply:
I would also add the explanation on what is different between each constructor besides the API. Possibly highlighting the fact that when externally controlled, the user has absolute control when the game lives and dies (and [who tells their story](https://www.youtube.com/watch?v=_gnypiKNaJE)), while when `.controlled` is used, the life of the game will be tied to the lifecycle of the widget (Remove the widget from the tree, kill the game).